lighthouse-cli/compiled-check.js

```diff
@@ -6,7 +6,7 @@ const path = require('path');
 module.exports = function(filename) {
   if (!fs.existsSync(path.join(__dirname, filename))) {
     console.log('Oops! Looks like the CLI needs to be compiled. Please run:');
-    console.log('   cd lighthouse-cli && npm install && npm run build');
+    console.log('   npm run install-cli');
```

Comment:
Right. I thought it would still be a good idea to have this around until that happens...given our history :\